test_scripts/Policies/build_options/090_ATF_For_PTU_use_consented_device_only.lua

```diff
@@ -18,165 +18,86 @@
 ---------------------------------------------------------------------------------------------------------------------
 require('user_modules/script_runner').isTestApplicable({ { extendedPolicy = { "EXTERNAL_PROPRIETARY" } } })
 
---[[ General Settings for configuration ]]
-Test = require('user_modules/connecttest_resumption')
-require('cardinalities')
-local mobile_session = require('mobile_session')
-local mobile_adapter_controller = require("mobile_adapter/mobile_adapter_controller")
-local file_connection = require('file_connection')
-local mobile = require('mobile_connection')
-local events = require('events')
-local utils = require ('user_modules/utils')
-
---[[ General configuration parameters ]]
--- Create dummy connection
-os.execute("ifconfig lo:1 1.0.0.1")
-
 --[[ Required Shared libraries ]]
-local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
-local commonSteps = require('user_modules/shared_testcases/commonSteps')
-local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
-require('user_modules/AppTypes')
-
---[[ General Precondition before ATF start ]]
-commonFunctions:SDLForceStop()
-commonSteps:DeleteLogsFileAndPolicyTable()
-
---[[ Local variables ]]
-local mobileHost = "1.0.0.1"
-local deviceMAC2 = "9cc72994ab9ca68c1daaf02834f7a94552e82aad3250778f2e12d14afee0a5c6"
-local deviceName2 = mobileHost .. ":" .. config.mobilePort
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/TheSameApp/commonTheSameApp')
+local common2 = require('test_scripts/Capabilities/PersistingHMICapabilities/common')
```

Comment:
```suggestion

--[[ Test Configuration ]]
runner.testSettings.isSelfIncluded = false
```
common2 shouldn't be needed, I think the issue is that `isSelfIncluded = false` is missing.